ts/navigation/AppNavigator.ts

```diff
@@ -15,7 +15,7 @@ const navigator = SwitchNavigator({
   [ROUTES.INGRESS]: {
     // This screen check if the user is authenticated than perform a redirect to
     // MainNavigator (if authenticated) or AuthenticationNavigator (otherwise)
-    screen: IngressScreen
+    screen: WalletHomeScreen // WIP to be replaced by IngressScreen before merging w/ master
```

Comment:
is this still wip?

Reply:
Reverting this to the original `IngressScreen` and adding comments to the classes are the two things I mentioned in the TODO list so far, will take care of those asap